rest_api/overview.adoc

```diff
@@ -4,8 +4,11 @@
 :data-uri:
 :icons:
 :experimental:
+:toc: macro
+:toc-title:
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+The OpenShift distribution of Kubernetes includes the link:./kubernetes_v1/overview.html[Kubernetes v1 REST API]
+and the link:./openshift_v1/overview.html[OpenShift v1 REST API]. These are RESTful APIs accessible via HTTP(s)
+on the OpenShift master servers.
```

Comment:
Did you want these to link to the other two new topics? If so they can be `link:./kubernetes_v1.html` and `link:./openshift_v1.html`, respectively.